src/basic/fsverity-util.h

```diff
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#pragma once
+
+#if HAVE_LINUX_FSVERITY_H
```

Comment:
i think it would be better to just copy fsverity.h into src/shared/linux/ like we do for some others.

Reply:
seems overkill, we don't need anything particularly new, the oldest one is good enough